### ISSUE TITLE
Add multiple language support

### ISSUE THREAD

**Issue by george112n** [BTEUK/TeachingTutorials#86]
Needs the ability for the language to be changed by a player.

There preference must be stored somewhere so they don't have to reselect their language when they rejoin the server.

**Comment by DonatiFilippo** [comment]
This would require a lot of work tho, as LangLibs ain't aimed as such dynamic use-case

**Comment by george112n** [comment]
For menus etc it would be fine, though for tutorials themselves it probably wouldn't, we'd probably need our own implementation.

**Comment by DonatiFilippo** [comment]
Yeah, well, actually as the lib is being rework to accommodate stuff for BTT, this could also be inserted. I'll consider this a bit
